packages/client/src/arpa_reporter/views/Home.vue

```diff
@@ -143,7 +140,7 @@ export default {
       this.sending = true;
 
       try {
-        const result = await getJson('/api/exports?async=true');
+        const result = await getJson('/api/exports?queue=true');
```

Comment:
This new query-parameter is being managed here:

https://github.com/usdigitalresponse/usdr-gost/pull/2095/files#diff-badaddb12479d69a9f44064517af0399f9aea8079c4458a9380a90d11999c87fR59